hatchways/team-bluejay
# In chef profile

To test:

- create user
- check profile page. edit stuff
- "become a chef"
- check profile page, should now be a chef's profile. edit stuff
- go to "/chefs/<id>" edit buttons should not be showing if viewing your own page. 

Also, try playing around submitting an existing user, logging in with a wrong password. Alert should be showing on the screen with descriptive messaging
